flink-quickstart/flink-quickstart-java/src/main/resources/archetype-resources/pom.xml

```diff
@@ -128,6 +128,7 @@ under the License.
 							<goal>shade</goal>
 						</goals>
 						<configuration>
+							<dependencyReducedPomLocation>${project.basedir}/target/dependency-reduced-pom.xml</dependencyReducedPomLocation>
```

Comment:
I was rather thinking of setting `<createDependencyReducedPom>false</createDependencyReducedPom>` because for applications this pom isn't usually required.

Reply:
agree, thanks for the hint
done